configure.ac

```diff
@@ -226,13 +227,8 @@ AC_ARG_ENABLE([defaultflags],
               [enable_defaultflags=yes])
 AS_IF([test "x$enable_defaultflags" = "xyes"],
       [
-      ADD_PREPROC_FLAG([-D_DEFAULT_SOURCE])
-      ADD_PREPROC_FLAG([-D_BSD_SOURCE])
-      ADD_PREPROC_FLAG([-D_POSIX_SOURCE])
       AS_IF([test "x$enable_debug" = "xno"],
```

Comment:
`_POSIX_SOURCE` was added in #721 for a similar reason as `_BSD_SOURCE`. It is [also included in `_GNU_SOURCE`](https://github.com/tpm2-software/tpm2-tss/pull/721#discussion_r162766113), so dropping it in favour of `AC_USE_SYSTEM_EXTENSIONS` should be fine.